src/components/header/Header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+	width: 100%;
+	height: 60px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	box-shadow: 0 -2px darkgray inset
+}
+
+.logo {
+	padding: 10px;
+	height: 100%;
+	font-size: 32px;
+	text-align: center;
+	font-weight: 700;
+	display: flex;
+	align-items: center;
+
+	&__img {
+		height: 32px;
+		margin: 0;
+		padding: 0;
+	}
+}
+
+.user {
+	display: flex;
+	align-items: center;
+	gap: 10px;
+
+	&__avatar {
+		height: 30px;
+		width: 30px;
+		border: 1px solid black;
+		border-radius: 100%;
+		padding: 5px;
+	}
+
+	&__name {
+		font-size: 18px;
+		font-weight: bold;
+	}
+}
+
+@media screen and (max-width: 478.98px) {
```

Comment:
why such a weird screen size ? Checkout the media Queries for Standard Devices

Reply:
I used a responsive design approach to creating styles, so I didn't use media queries for Standard Devices. And **.98px is a rule that has become a habit for me, in order to avoid cross-over styles (styles overlapping) in the future, in case of: ```@media screen and (max-width: 768px)``` && ```@media screen and (min-width: 768px)``` when our viewport is really has a width 768px.  I think this is a useful rule, but if the command says to follow the standard media queries, then Ok, I'm fine with it 🙄 By the way, you can read about it [here](https://stackoverflow.com/questions/54596219/why-do-some-max-width-media-queries-use-a-98px)